misk/web/@misk/components/src/components/TextHTMLOrElementComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react"
+
+/**
+ * <TextHTMLOrElementComponent content={<div>item</div>} length={35} />
+ * <TextHTMLOrElementComponent content={"Test"} length={35} />
+ * <TextHTMLOrElementComponent content={'<a href="http://squareup.com/">Test</a>'} length={35} />
+ *
+ * Renders passed in text, HTML-parseable string, or React element with optional length constraints for text.
+ */
+
+export interface ITextHTMLOrElementProps {
+  content: string | Element | JSX.Element
+}
+
+export const TextHTMLOrElementComponent = (
+  props: ITextHTMLOrElementProps
+): JSX.Element => {
+  const { content }: any = props
```

Comment:
Can this be `const { content }: string | Element | JSX.Element`?

Reply:
Thanks! That's a good catch!